docs/plugins/inputs/file.asciidoc

```diff
@@ -410,7 +410,7 @@ The sincedb record now has a last active timestamp associated with it.
 If no changes are detected in a tracked file in the last N days its sincedb
 tracking record expires and will not be persisted.
 This option helps protect against the inode recycling problem.
-Filebeat has a {filebeat-ref}/faq.html#inode-reuse-issue[FAQ about inode recycling].
```

Comment:
The Filebeat FAQ rework wasn't backported to 6.8, and the "updated" link won't work.